docker-compose.test.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
+  db:
```

Comment:
Maybe it's clear if service name refs to `db-test` or something similar to avoid confusions with development `db`